superset-frontend/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -107,21 +107,33 @@ class ChartHolder extends React.Component {
       outlinedComponentId: null,
       outlinedColumnName: null,
       directPathLastUpdated: 0,
+      isFullSize: false,
+      resizeEvent: false,
     };
 
     this.handleChangeFocus = this.handleChangeFocus.bind(this);
     this.handleDeleteComponent = this.handleDeleteComponent.bind(this);
     this.handleUpdateSliceName = this.handleUpdateSliceName.bind(this);
+    this.handleFullSize = this.handleFullSize.bind(this);
   }
 
   componentDidMount() {
     this.hideOutline({}, this.state);
   }
 
   componentDidUpdate(prevProps, prevState) {
+    this.finishResizeEvent();
     this.hideOutline(prevState, this.state);
   }
 
+  finishResizeEvent() {
+    setTimeout(() => {
+      if (this.state.resizeEvent) {
+        this.setState(() => ({ resizeEvent: false }));
+      }
+    }, 200);
```

Comment:
are we guaranteed that the event will end in 200ms? what if something is really slow?

Reply:
resizeEvent deleted b79ed4fc5348742c102b9e5313669749eb699700, per @suddjian suggestion